client/state/followers/test/selectors.js

```diff
@@ -57,7 +49,49 @@ describe( 'selctors', () => {
 				},
 			};
 			const query = { siteId: 555 };
-			expect( getFollowersForQuery( state, query ) ).to.be.null;
+			expect( getFollowersForQuery( state, query ) ).toBeNull;
+		} );
+	} );
+
+	describe( '#getIsFetchingFollowersForQuery', () => {
+		test( 'should return true if currently fetching', () => {
+			const serializedQuery = getSerializedQuery( { siteId: 555 } );
+			const state = {
+				followers: {
+					queryRequests: {
+						[ serializedQuery ]: true,
+					},
+				},
+			};
+			expect( getIsFetchingFollowersForQuery( state, { siteId: 555 } ) ).toBe( true );
+		} );
+
+		test( 'should return false if not currently fetching', () => {
+			const serializedQuery = getSerializedQuery( { siteId: 555 } );
+			const state = {
+				followers: {
+					queryRequests: {
+						[ serializedQuery ]: false,
+					},
+				},
+			};
+			expect( getIsFetchingFollowersForQuery( state, { siteId: 555 } ) ).toBe( false );
+		} );
+	} );
+
+	describe( '#getTotalFollowersForQuery', () => {
+		test( 'should return the total number of followers for a query', () => {
```

Comment:
Do we need a test to cover the case when there is no data for that query?

Reply:
fixed via 31043440f5